charts/redpanda/post_install_upgrade_job.go

```diff
@@ -97,18 +97,18 @@ func PostInstallUpgradeJob(dot *helmette.Dot) *batchv1.Job {
 			Name:      fmt.Sprintf("%s-configuration", Fullname(dot)),
 			Namespace: dot.Release.Namespace,
 			Labels: helmette.Merge(
-				FullLabels(dot),
 				helmette.Default(map[string]string{}, values.PostInstallJob.Labels),
+				FullLabels(dot),
```

Comment:
For this change if `PostInstallJob.Labels` is considered an **override** then shouldn't we allow for **overriding** `FullLabels(dot)`?

Reply:
This change does exactly that. It allows for overriding. I will add test case to assert that generated label by the `FullLabel` function will be changed if I provide in the same key different label value.